api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -255,6 +255,8 @@ public String freeTierBillingAccountName() {
     public String queueName;
     // Number of ids per task
     public Integer exportObjectsPerTask;
+    // Should workspace details be excluded for researcher directory
+    public boolean excludeFromPublicDirectory;
```

Comment:
nit: I know this is verbose but could this be called `excludeWorkspaceDetailsFromPublicDirectory` or something that makes it clear that it's only workspace details that are being excluded?

Reply:
thats a good idea (they may come back with another flag for researcher it will be easy to differentiate )